### ISSUE TITLE
keyboard focus not visible well enough

### ISSUE THREAD

**Issue by Soardiac** [swiss/styleguide#254]
![a4a2 2p12](https://cloud.githubusercontent.com/assets/7532362/4441351/c4a61044-47d1-11e4-8856-8c8529df35d1.PNG)

this is from A4A's retest report, chapter 2.4, page 12
